devices.js

```diff
@@ -4170,6 +4170,15 @@ const devices = [
         fromZigbee: [fz.ias_contact_alarm_1],
         toZigbee: [],
     },
+    {
+        zigbeeModel: ['902010/24A'],
+        model: 'AV2010/24A',
+        vendor: 'Bitron',
+        description: 'Optical Smoke Detector',
+        supports: 'tamper',
+        fromZigbee: [fz.bitron_zone_status],
```

Comment:
Can this be replaced with `fz. ias_smoke_alarm_1`?

Reply:
We could but should then extend `fz. ias_smoke_alarm_1`. The current implementation doesn't parse i.e. alarm_2 and I'm not sure if this is used by the smoke detector. I know that the smoke detector can also be used as an alarm system and therefore it might be useful in the future.

Anyway, is there any reason why the current implementation doesn't parse all values from the message, even if not used?

Reply:
Besides `alarm_2` I'm fine adding all the other attributes to `ias_smoke_alarm_1`.

About `alarm_2`, I've only seen device which either use alarm 1 or 2 (not both), therefore it doesn't make sense to provide both values (because one will always be false), it makes it harder for people to understand which value to use.

Reply:
I've read a bit in the ZCL specification: https://etc.athom.com/zigbee_cluster_specification.pdf

In 8.2.2.2.1.2:
> The ZoneType attribute values are summarized in Table 8.5. The Zone Type dictates the meaning of Alarm1 and Alarm2 bits of the ZoneStatus attribute, as also indicated in this table. 

The table shows a lot of devices that support two alarm types. For the device at hand this does not apply since it identifies itself as a 0x0028 Fire Sensor that only uses Alarm1.

Shall I update the PR or just issue a clean new one?

Reply:
Just update the PR, I will squash the PR anyway.